dune

```diff
@@ -42,3 +42,20 @@
  (name runtest)
  (package coqide-server)
  (deps test-suite/summary.log))
+
+; This is an ugly hack, we should use
+; %{lib_root:coq/theories/Init/Prelude.vo} when Dune supports it.
```

Comment:
When will dune support it?

Reply:
Likely next version, but indeed this PR may be closed in favor of the native solution before.